rosetta-2021.html

```diff
@@ -0,0 +1,236 @@
+<script>
+  // Rule of thumb: put a semicolon at the end of every line that doesn't end in a curly
+
+  // Simple output
+  // =============
+
+  // puts "howdy!"
+
+  // alert("howdy!");
```

Comment:
`alert` is definitely the visible way to see output of JS in the browser. In AppDev 1, we specify that not to use `puts` in Rails projects since it prints to the Terminal. 

I think a more accurate relationship would be with `console.log` instead of `alert` since that prints to the console and works outside of browsers.